fabric-transfer-api-v1/src/main/java/net/fabricmc/fabric/api/transfer/v1/transaction/TransactionResult.java

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright (c) 2016, 2017, 2018, 2019 FabricMC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package net.fabricmc.fabric.api.transfer.v1.transaction;
+
+public enum TransactionResult {
+	ABORTED,
+	COMMITTED;
+
+	public boolean wasAborted() {
```

Comment:
was -> is

Reply:
it's a result, `was` makes sense

Reply:
Past tense isn't really going to make sense. The transaction is canceled.

Reply:
Imo we should just move it to a boolean. If this enum will have more entries this defeats the point of using an enum vs a pseudoenum that allows extension

Reply:
When it is called, the transaction is already closed. Therefore I think it makes more sense to use a past tense.

Reply:
Past tense for transaction doesn't mean past tense for the result. The result ***is*** always there since it ***has been*** or ***was*** derived.

Reply:
I don't think should be a boolean. It prevents errors (e.g. wrongly naming the boolean `successful` instead of `failed` is very hard to troubleshoot) by making the API so much more explicit, and it's really a trivial class. I am against result `boolean`s in general for those reasons.

Reply:
We usually use true for success and false for failure/voided stuff. So much that

Reply:
This is based on my experience, when I was implementing the API I didn't remember if the `boolean` was for success or failure. A simple `result.wasAborted()` is so much more expressive. We are talking about a 30 LOC helper enum... I really think booleans should only be used for conditions, and not as flags...